src/axom/sidre/core/DataStore.hpp

```diff
@@ -88,6 +88,21 @@ class DataStore
    */
   const Group* getRoot() const { return m_RootGroup; };
 
+  /// Return whether a Conduit error occurred.
+  bool getConduitErrorOccurred() const { return m_conduit_error_occurred; };
+  void setConduitErrorOccurred(bool flag) const
+  {
+    m_conduit_error_occurred = flag;
+  };
+
+  /// Return information on any Conduit errors.
+  std::string getConduitErrors() const { return m_conduit_errors; };
+  void clearConduitErrors() const { m_conduit_errors.clear(); };
+  void appendToConduitErrors(const std::string& mesg) const
+  {
+    m_conduit_errors = m_conduit_errors + "\n" + mesg;
+  };
```

Comment:
Would a `std::list<std::string>`  (or similar) be better than appending strings on the fly?

A benefit would be that DataStore would not need to explicitly manage a separate `bool` for `m_conduit_error_occurred` -- it could instead be `m_conduit_errors.empty()`.